### DISCUSSION TITLE
Polaroid effect, but no warp or rotate

### DISCUSSION THREAD

**Discussion by RrnR** [ImageMagick/ImageMagick#6123] [Help] [answered]
### ImageMagick version

7.1.0-43 Q16-HDRI x64 c95ef31:20220709

### Operating system, version and so on

Windows 10

### Description

I want the polaroid effect, but without the warping of the image and drop shadow, and no rotation.  I can remove the rotation by using `-polaroid 0`, but can I remove the warping somehow (essentially I still want the border and drop shadow)?

**Comment by snibgo** [comment, marked as answer]
You can just add a border and a drop shadow.

See https://imagemagick.org/script/command-line-options.php#border and https://imagemagick.org/Usage/blur/#shadow
